vendors/pc/boards/linux/CMakeLists.txt

```diff
@@ -2,10 +2,18 @@ if (AFR_ENABLE_UNIT_TESTS)
 
     set(CMAKE_C_FLAGS_DEBUG "-ggdb3 -Og")
     set(CMAKE_C_FLAGS_RELEASE "-O3")
-    set(CMAKE_C_FLAGS "-Wall")
+    set(CMAKE_C_FLAGS "-Wall -Wextra -Wpedantic")
```

Comment:
Does this add these warnings to the mocks as well? That might create a lot of noise when compiling

Reply:
Yes it does actually, I will fix that